lib/matplotlib/__init__.py

```diff
@@ -192,7 +192,17 @@ def _check_versions():
             ("numpy", "1.11"),
             ("pyparsing", "2.0.1"),
     ]:
-        module = importlib.import_module(modname)
+        try:
+            module = importlib.import_module(modname)
```

Comment:
Is Kiwisolver the only module for which this is true?  In which case, maybe check `modname` to avoid confusing people who get import errors from other modules.

Reply:
Probably not, but it is the only one I've experienced the problem with. 

Also if I'm not mistaken, kiwisolver is not even used by most matplotlibusers. It plays no role in creating basic scatter plots, histograms etc., does it? Maybe one matplotlib could just issue a warning about kiwisolver not being available and load anyway. Someone suggested something like it once: https://github.com/matplotlib/matplotlib/issues/10386